### ISSUE TITLE
Error whenever I change subtitles on an episode

### ISSUE THREAD

**Issue by BEisem** [RemiRigal/Plex-Auto-Languages#4]
I'm trying to get Plex-Auto-Languages to work with my setup. I installed via Docker. Every time I turn on subtitles on an episode, the Docker logs show a crash/traceback. Here is the traceback:

```
2022-04-22 19:08:23,227 [ERROR] Unable to process play session
Traceback (most recent call last):
  File "main.py", line 57, in process_playing_message
    self.process_play_session(play_session)
  File "main.py", line 93, in process_play_session
    self.change_default_tracks_if_needed(item)
  File "main.py", line 145, in change_default_tracks_if_needed
    changes = PlexUtils.get_track_changes(episode, episodes)
  File "/app/utils/plex.py", line 107, in get_track_changes
    matching_audio_stream = PlexUtils.match_audio_stream(selected_audio_stream, part.audioStreams())
  File "/app/utils/plex.py", line 72, in match_audio_stream
    if reference.title <= stream.title:
TypeError: '<=' not supported between instances of 'NoneType' and 'str'
```

I should mention, although I installed via Docker, my Plex install is native. So in the config.yaml file, I used the reverse proxy URL for my Plex server. I'm not sure if this is a valid configuration.

**Comment by mcpowell87** [comment]
I am having a very very similar issue except for the fact that the TypeError being thrown is `TypeError: '<=' not supported between instances of 'NoneType' and 'NoneType'`

Running docker plex and docker plex-auto-languages.

**Comment by RemiRigal** [comment]
The issue has been resolved and deployed in v1.0.3.
